src/checks/incorrect_delimiter.rs

```diff
@@ -0,0 +1,88 @@
+use crate::checks::{Check, Warning};
+use crate::LineEntry;
+
+pub(crate) struct IncorrectDelimiterChecker {}
```

Comment:
This struct should have the field `warning: Warning`.
Example: https://github.com/mgrachev/dotenv-linter/blob/master/src/checks/leading_space.rs#L5